React/game/src/app/components/Screen/Game/Square/index.js

```diff
@@ -0,0 +1,19 @@
+import React, { Component } from 'react';
```

Comment:
Same about @DLotero says. Following this file structure rules -> https://github.com/Wolox/tech-guides/blob/master/react/style-guide.md#folder-structure Move this subcomponent to folder `.../screens/Game/components/Square/...`. Also, `screens` folder must be in minus

Reply:
👍

Reply:
@unidrelyn Check again this. This change wasn't applied